plugins/dev-tools/src/addGUIControls.js

```diff
@@ -90,6 +89,10 @@ export default function addGUIControls(createWorkspace, defaultOptions) {
     onChangeInternal();
   };
 
+  // Options folder.
```

Comment:
This function is huge--please break it up into helper functions.

Reply:
Split up.